Website/src/rendering/src/assets/css/components/Speakers/SpeakersGrid.css

```diff
@@ -1,59 +1,38 @@
-.section__speakers__grid {
-  @apply mx-auto grid grid-cols-1 gap-5;
-  @apply sm:grid-cols-2;
-  @apply md:grid-cols-3;
-  @apply lg:grid-cols-4;
-  @apply xl:grid-cols-5;
-
-  margin-top: 20px;
-  margin-bottom: 40px;
-
-  &__speaker {
-    @apply box-border col-span-1 relative;
-
-    width: 100px;
-    min-width: 260px;
-    padding: 10px;
-    margin: 10px;
-
-    &:hover img {
-      box-shadow: 0 0 10px rgba(23, 23, 23, 0.45);
-    }
-
-    img {
-      @apply w-full;
-    }
+.section__speakers {
+  .section__content__title {
```

Comment:
This CSS class is not used anymore in this component. It can be deleted.

Reply:
The `section__speakers` CSS class is used at 2 different locations for 2 different things:
1. As the "/sitecore/content/EdgeWebsite/home/Page Components/Speaker Line-up Section" item cssClass field value for a home page Section component to have a specific background and title style.
2. In the `SpeakersGrid.tsx` component. This component has no background image nor title to display.

I suggest:
1. We move the `section__speakers` and its title classes to a different file `/layout/section-speakers.css`.
2. In the `SpeakersGris.css` and TSX files, we rename the top-level CSS class to `speakers-grid` to match the other component CSS files naming.

Reply:
its speaker specific section that you moved under speakers folder. I think this makes sense.